secretmanager/api/SecretManager.Samples.Tests/CreateUmmrSecretTests.cs

```diff
@@ -0,0 +1,41 @@
+/*
```

Comment:
I don't like the name "UMMR" in the filename. Can we say "UserManaged" instead?

Reply:
updated. using "UserManagedReplicationSecret"